### ISSUE TITLE
[Feature Request] Dynamic Trees Support

### ISSUE THREAD

**Issue by Spazmunki13** [progwml6/Natura#441]
I'd love to see your trees work the Dynamic Trees mod.

**Comment by Dalethium** [comment]
Same! I absolutely love that mod, and this mod! Perhaps looking at how biomes-o-plenty did their compat patch may be fruitful?

**Comment by nephatrine** [comment]
Would be a terrific idea. Might make eucalyptus trees not such a chore to chop down anymore too as a side effect.

**Comment by Pulz-TCP** [comment]
I hope this suggestion hasn't been forgotten about. I'd love to see all my favourite mods (especially natura) have compatibility with Dynamic Trees!

**Comment by Graywaren** [comment]
Me Too!!!

**Comment by nephatrine** [comment]
Yeah this is the last mod I'm still waiting on to get support. Figure I'm going to have to learn how to make my own DT addon sometime and do it myself.

**Comment by stormbringerGT** [comment]
Agreed, our server is based around dynamic trees so seeing a compatibility patch would be terrific!

**Comment by Azuriuz** [comment]
I want this in my pack but it'd ruin the immersion since it doesn't work with Dynamic Trees; please please please put out support, or an add-on for it?

**Comment by lolsteamroller** [comment]
Yes, me too, currently using it for my hardcore survival pack, Natura support would be really really nice.

**Comment by supermassimo** [comment]
Everyone: I have been working on tackling this project myself!
It is still WIP so there isnt worldgen, but all the trees are accessible through the dirt bucket recipe.
Check it out:
(If it is unavailable its because it is still processing, check in a couple hours)
https://www.curseforge.com/minecraft/mc-mods/dynamic-trees-natura
